src/AI/agents/agents/graph/runner.py

```
@@ -194,21 +233,8 @@ async def run_once(state: AgentState, event_sink: EventSink = None):
-                            await run_intent_judge(
-                                user_goal=state.user_goal,
-                                agent_plan=step_plan[0] if isinstance(step_plan, list) and step_plan else None,
-                                trace_id=root_span.trace_id,
-                            )
-                        except asyncio.CancelledError:
-                            raise
-                        except Exception:
-                            log.exception("Evaluation pipeline error")
-
-                    asyncio.create_task(_run_intent_match_evaluation())
+                    asyncio.create_task(_evaluate_intent_match(state.user_goal, final_state, root_span.trace_id))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)